client/components/learn/Resources/resources-model.ts

```diff
@@ -232,6 +232,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://github.com/getify/You-Dont-Know-JS',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/ydkjs_500w.png`,
     },
+    {
+      title: 'Patterns Dev',
+      url: 'https://patterns.dev/',
+      cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/patterns.dev.png`,
```

Comment:
Here you have `patterns.dev.png` but the actual file name should be `patterns-dev_500w.png` (please rename the png file. You have a typo. Should be **patterns-dev_500w.png**.)